mods/d2k/weapons.yaml

```diff
@@ -5,8 +5,8 @@ LMG:
 	Projectile: Bullet
 		Speed: 1c256
 	Warhead@1Dam: SpreadDamage
-		Spread: 48
-		Falloff: 100, 100, 65, 30, 0
+		Spread: 96
+		Falloff: 100, 50, 25, 12, 6, 0
```

Comment:
Is there a specific reason why you didn't go back to our default falloff model for these? (the two are basically equal if you scale the spread by a factor of ~1.4 and don't mind the wider negligible-damage tail).